vignettes/safestats-vignette.Rmd

```diff
@@ -224,16 +222,12 @@ To see the distributions of stopping times of only the experiments where the nul
 plot(simResultDeltaTrueIsDeltaMin, showOnlyNRejected=TRUE)
 ```
 
-Hence, the simulations demonstrate how power is gained as a result of optional stopping whenever the true effect size equals the minimal clinical relevant effect size. 
-
 ### Safe tests detect the effect early if it is present: deltaTrue larger than deltaMin
-<!--#### Optional stopping: True effect size larger than the minimal clinical relevant effect size-->
+<!--#### Optional stopping: True effect size larger than the minimal clinically relevant effect size-->
 What we believe is clinically minimally relevant might not match reality. One advantage of safe tests is that they perform even better, if the true effect size is larger than the minimal clinical effect size that is used in the planning of the experiment. To see this, we run the following code
 
 ```{r}
-# alpha <- 0.05
```

Comment:
The alpha, beta and deltaMin are defined above, but I wanted to keep them in the code, so people know what their value is. Perhaps my intention wasn't clear and we should add something such as

    # Recall:
    # alpha <- 0.05
    # beta <- 0.2
    # deltaMin <- 9/(sqrt(2)*15)      # = 0.42